mobile/app/src/main/java/org/exarhteam/iitc_mobile/prefs/PluginPreferenceActivity.java

```diff
@@ -343,7 +350,7 @@ private void addHeader(final String title, final boolean userPlugin) {
         final Header newHeader = new Header();
         newHeader.title = title;
         newHeader.fragmentArguments = bundle;
-        newHeader.fragment = "org.exarhteam.iitc_mobile.fragments.PluginsFragment";
+        newHeader.fragment = "org.exarhteam.iitc_mobile_px.fragments.PluginsFragment";
```

Comment:
I already wrote above that it is not required

Reply:
Yep will change it back